server/lib/changelly/changelly.ctrl.js

```diff
@@ -43,4 +43,24 @@ ChangellyCtrl.getCurrencies = (req, res) => {
   })
 }
 
+ChangellyCtrl.getMinAmount = (req, res) => {
+  return ChangellyCtrl.request('getMinAmount', req.params, (err, data) => {
+    if (err) {
+      console.log('Error: ', err)
```

Comment:
if there's an error we probably need to res.send() something back to the front end telling it that there was an API error otherwise this request would just hang until server timeout.